railties/test/commands/credentials_test.rb

```diff
@@ -310,6 +329,23 @@ def write_credentials(content, **options)
       end
     end
 
+    def assert_credentials_paths(content_path, key_path, environment: nil)
+      content = "foo: #{content_path}"
+      remove_file content_path
+      remove_file key_path
+
+      write_credentials content, environment: environment
+      assert_file content_path
+      assert_file key_path
+
+      assert_match content, run_show_command(environment: environment)
+
+      # Decrypted diffs apply to credentials files in standard locations only.
```

Comment:
This seems reasonable, but is probably worth a note in the changelog.

Reply:
This has not changed.  `bin/rails credentials:diff --enroll` already behaves this way.  Here, I am merely explaining the reason for the conditional assertion.

Reply:
Yeah so the changelog may not be the right home. But I think we should document somewhere that `content_path` don't work with diffs, if it's not already documented.